packages/commonwealth/client/scripts/views/components/chat/chat_window.tsx

```diff
@@ -37,17 +38,50 @@ type ChatWindowAttrs = {
 };
 
 export class ChatWindow implements m.Component<ChatWindowAttrs> {
-  private onIncomingMessage: (any: any) => void;
-  private scrollToBottom: () => void;
-  private shouldScroll: boolean;
-  private shouldScrollToHighlight: boolean;
-  private quillEditorState: any;
+  onIncomingMessage: (any: any) => void;
+  scrollToBottom: () => void;
+  shouldScroll: boolean;
+  shouldScrollToHighlight: boolean;
+  quillEditorState: QuillEditor;
+  channel;
```

Comment:
Type?

Reply:
Nobody knows! Not my code, tried to figure out